Sourcery/Parsing/Parser.swift

```diff
@@ -129,6 +149,13 @@ final class Parser {
             } else {
                 containingType.variables += [variable]
             }
+
+        if let enumeration = containingType as? Enum {
```

Comment:
If I understand correctly `processContainedType` will be called for any variable defined inside the type, so we might do the same work again and again. My initial intention was also to put it here, but then I moved it. I think at least this block can be moved to `else` brunch of previous `if` as we know for sure that `rawValue` is not a static variable.
Apart from that looks good! 👍